sdk/storage/azure-storage-file/CHANGELOG.md

```diff
@@ -1,14 +1,33 @@
 # Release History
 
-## 12.0.0-preview.2 (2019-08-08)
+## Version 12.0.0-preview.3 (2019-09-10):
+For details on the Azure SDK for Java (September 2019 Preview) release, you can refer to the [release announcement](https://aka.ms/azure-sdk-preview3-java).
+
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-file_12.0.0-preview.3/sdk/storage/azure-storage-file/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-java/tree/azure-storage-file_12.0.0-preview.3/sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file)
+demonstrate the new API.
+
+- Added tracing telemetry on maximum overload API.
+- Added generate SAS token APIs.
+- Throw `StorageException` with error code when get error response from service.
+- Moved `ReactorNettyClient` into a separate module as default plugin. Customer can configure a custom http client through builder.
+- Throw `UnexpectedLengthException` when the upload body length does not match the input length.
+- Added validation policy to check the equality of request client id between request and response.
+- Added `PageFlux` on async APIs and `PageIterable` on sync APIs.
+
```

Comment:
Upgraded to use service version 2019-02-02 instead of 2018-11-09

Reply:
Done.